documentation/src/main/asciidoc/topics/contributing/terminology.adoc

```diff
@@ -0,0 +1,86 @@
+[[terminology]]
+= {brandname} Terminology
+
+{brandname} terminology refers to words and phrases specific to {brandname}
+capabilities and features. It is important to use consistent terminology across
+documentation and the code base to avoid unclear or contradictory language.
+
+Add::
+Add an item to an existing list, group, view, or other container element. For example, "add entries to caches".
+
+Cache configuration::
+An XML or JSON document that controls {brandname} behavior.
+
+Cache template::
+An artifact from which you create cache configurations. +
+{brandname} provides some pre-defined templates that can be referenced using the `org.infinispan.client.hotrod.DefaultTemplate` enum. +
```

Comment:
`org.infinispan.client.hotrod.DefaultTemplate` is only for Hot Rod. I would mention that the REST and Server config could use the name instead. Example: `org.infinispan.DIST_SYNC`
It would be nice to have a link to DefaultTemplate.